plugin-tester-java/build.sbt

```diff
@@ -6,12 +6,16 @@ enablePlugins(AkkaGrpcPlugin)
 
 javaAgents += "org.mortbay.jetty.alpn" % "jetty-alpn-agent" % "2.0.7" % "runtime"
 
+//#protoSources
 inConfig(Compile)(Seq(
+  PB.protoSources += new File("src/main/proto"),
+  //#protoSources
```

Comment:
Hmm. I'd want this to be just `PB.protoSources += ...` also, I think the sbt docs says you should _never_ use relative paths but always base of off the sources key or such.

Reply:
That makes sense, let me give that a quick try